cmd/cluster/aws/create.go

```
@@ -252,8 +260,18 @@ func IsRequiredOption(flag string, value string) error {
 // not empty; validates if the credentials secret is not empty, that it can be retrieved
 func ValidateCreateCredentialInfo(opts *core.CreateOptions) error {
 	if len(opts.CredentialSecretName) == 0 {
-		if err := IsRequiredOption("aws-creds", opts.AWSPlatform.AWSCredentialsFile); err != nil {
-			return err
+		if opts.AWSPlatform.AWSCredentialsFile == "" {
```

We should throw an error if both `--aws-creds` and `--sts-creds` are passed